code/app/src/main/java/com/example/sigma_blue/entity/item/Item.java

```diff
@@ -436,7 +449,8 @@ public int hashCode() {
                     q.getString("MAKE"),
                     q.getString("MODEL"),
                     q.getString("SERIAL"),
-                    Float.parseFloat(q.getString("VALUE"))
+                    Float.parseFloat(q.getString("VALUE")),
+                    q.getString("IMAGE")
```

Comment:
This is using an old version of the Item class. Please rebase/squash development into this branch, and then make a new account and test it. The latest version of the system no longer uses HashMap<String, String>, and instead uses HashMap<String, Object>. This means that there may be some conflicts when merging this into dev.

Reply:
I see the issues happened here. going to adapt the new item class, and trying to use similar structure in develop branch for multi-photo implementation, hopefully finishing tonight.